docs/source/color_processing/color-processing-introduction/color-processing-introduction.rst

```diff
@@ -4,12 +4,12 @@ FTC Color Processing - Introduction
 Overview
 --------
 
-As of FTC SDK 10.1, the *FIRST* Tech Challenge Software Development Kit now
-includes some Color Processing features from OpenCV, a popular and powerful
+The *FIRST* Tech Challenge Software Development Kit will soon
+include some Color Processing features from OpenCV, a popular and powerful
```

Comment:
I'm not a fan of "will soon" language, nor using future tense.  In general most places I've worked have had policies forbidding this language in documentation, design/requirements docs, etc.  The reason for this is that text that uses that has an immediate expiration date.  

It is best to avoid both "will" and "soon"

Can we change this to "The next release, 10.1, of the *FIRST* Tech Challenge Software Development Kit contains new color blob locating software for the VisionPortal."